migration.md

```diff
@@ -1,10 +1,14 @@
+# Note
+
+Users of `cortex-python` should install `sensa-python` instead.
```

Comment:
I would suggest repurposing this file to only be a about migrating  `cortex-python` -> `sensa-python`, because I don't think keeping the migration from `cortex-client` is useful anymore (if `cortex-python` is dead, why mention its predecessor?).

Reply:
Also, we make sure to say that `cortex-python` is deprecated here (maybe also in the README?)